airflow/www/views.py

```diff
@@ -49,12 +48,12 @@
 from wtforms import SelectField, validators
 
 import airflow
-from airflow import configuration as conf
-from airflow import models, jobs
+from airflow.configuration import conf
```

Comment:
what are we doing here? L51 and L52 doesn't seem correct.

Reply:
This is also related to the comment above. I see at some places conflict with the name `conf`. For example it is in the import but also an argument in a method is called `conf`. I could not really fix this a better way without touching even more code.

If someone has an idea about this please let me know. I think this should be outside this PR.

Reply:
how about 
```
from airflow import configuration as conf

....
conf.conf
```

or 
```
from airflow import configuration 
configuration.conf something 
```
this applies to the first comment as I think `from airflow import configuration as conf, configuration` is not very ideal.  WDYT